ConwaysGameOfLife/Generator.cs

```diff
@@ -38,9 +38,9 @@ public Generator(Universe universe)
             
         }
 
-        public void DisplayUniverse(IOutput output, OutputFormatter formatter)
+        public void DisplayUniverse(IOutput output)

```

Comment:
Will the `Generator` be responsible for telling the `IOutput` to do other things as well e.g. output the prompts for user input? Have a think about what "components" we need to build a `Generator` and how they should exist in relation to the `Generator` i.e. what dependency injection will be used.